lib/upsert/connection/jdbc.rb

```diff
@@ -21,7 +22,7 @@ module Jdbc
       end.merge(
         'TrueClass'  => 'setBoolean',
         'FalseClass' => 'setBoolean',
-        'Fixnum'     => 'setInt',
+        'Fixnum'     => 'setInt'
```

Comment:
i leave the `,` on the last hash element just in case i add more later - that way it doesn't show up in the diff :)